### ISSUE TITLE
Add profile for sharks at Oilrig

### ISSUE THREAD

**Issue by WheteThunger** [WheteThunger/MonumentAddons#5]
Create an example profile which adds shark spawn points to oilrig monuments.

**Comment by m-hynek** [comment]
I have sharks on rigs  - 2 on large, 1 on small. I'll clean it up and share in PR

**Comment by WheteThunger** [comment]
Below is a profile I was working on before, which has 5 sharks at large Oil Rig. Fee free to reuse any of it for the profile you are making.

Some things to keep in mind:
- In practice, I found that multiple spawn points were needed to ensure it was likely that a player would run into a shark when swimming below Oil Rig
- The reward for killing sharks may be significant, especially on modded servers, so we don't want to go overboard with the shark respawn frequency
- It's much easier for a server owner to remove a spawn point from a profile that they installed than it is to add their own spawn point, so we should probably bias toward having more points than needed by default
- Sharks may be tough on server performance, so we probably want to keep the list of spawns relatively small, but note that sharks should sleep when players are not nearby to partially mitigate performance issues
- The shark patrol path is vanilla generated, but you can visualize it with plugins (just loop `shark.patrolPath` and use ddraw to plot some spheres and indices as text)
- Once #14 is done, we should be able to give people the ability to designate the shark patrol path

```json
{
  "Name": "OilrigSharks",
  "Author": "WhiteThunder",
  "SchemaVersion": 2.0,
  "MonumentData": {
    "OilrigAI": {
      "SpawnGroups": [
        {
          "Id": "b8c55194-6bbf-4245-8ceb-0763f804784c",
          "Name": "Sharks",
          "MaxPopulation": 5,
          "SpawnPerTickMin": 5,
          "SpawnPerTickMax": 5,
          "RespawnDelayMin": 1500.0,
          "RespawnDelayMax": 2100.0,
          "PreventDuplicates": false,
          "Prefabs": [
            {
              "PrefabName": "assets/rust.ai/agents/fish/simpleshark.prefab",
              "Weight": 100
            }
          ],
          "SpawnPoints": [
            {
              "Id": "efe3c661-2387-4ec7-aa02-f6ec2dd7f1eb",
              "Position": {
                "x": -10.0,
                "y": 0.0,
                "z": -17.0
              },
              "RotationAngles": {
                "x": 0.0,
                "y": 90.0,
                "z": 0.0
              },
              "Exclusive": true,
              "DropToGround": true
            },
            {
              "Id": "de248f4e-07bd-49fd-beb0-2706c199eced",
              "Position": {
                "x": 13.75,
                "y": 0.0,
                "z": 10.0
              },
              "Exclusive": true,
              "DropToGround": true
            },
            {
              "Id": "dd55b8e1-d6a1-408e-aff8-cdad51864b26",
              "Position": {
                "x": 35.0,
                "y": 0.0,
                "z": -16.0
              },
              "Exclusive": true,
              "DropToGround": true
            },
            {
              "Id": "2a9ef1c9-dd90-44ba-ab48-0e4ab8c01b7f",
              "Position": {
                "x": 13.75,
                "y": 0.0,
                "z": -40.0
              },
              "Exclusive": true,
              "DropToGround": true
            },
            {
              "Id": "296d6193-d25c-4d1e-b08e-fdcfb290e17a",
              "Position": {
                "x": 15.0,
                "y": 0.0,
                "z": -16.0
              },
              "Exclusive": true,
              "DropToGround": true
            }
          ]
        }
      ]
    }
  }
}
```

**Comment by WheteThunger** [comment]
Sounds good to me! Thanks for thinking it through.

I tested those example profiles and I like how evenly they are covering the area, as seen in the below screenshots.

![image](https://user-images.githubusercontent.com/7545246/165433751-da9f39ff-59c5-4d73-a971-a326abc04146.png)

![image](https://user-images.githubusercontent.com/7545246/165433892-b3f6aeea-bacd-4e35-9c62-9d05e06ffea2.png)

**Comment by WheteThunger** [comment]
Resolved in #34.
